playbook-website/app/views/pages/home.html.erb

```diff
@@ -20,4 +20,38 @@
   <%= render(MdHpSocialProofComponent.new) %>
   <%= render(SmHpSocialProofComponent.new) %>
   <%= render(HpSolutionsComponent.new)%>
+
+
+  <%= pb_rails("background", props: { background_color: "white" }) do %>
```

Comment:
Can this be make into a viewcomponent or a partial?

Reply:
Sure!

Reply:
Done! @jasoncypret